ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -213,10 +219,16 @@ Anyone who has access to this search ${formTitleLowerCase} will subsequently los
     )
   },
   handleClearDefault() {
```

Comment:
you will need to toggle this class https://github.com/codice/ddf/pull/5957/files#diff-1af92a3d1da92776b5779e1e855b8c24R114 if the current result form is default, otherwise the `clear default` option that calls this function will never show